internal/celext/lib.go

```diff
@@ -70,6 +71,47 @@ func (l lib) CompileOptions() []cel.EnvOption {
 			l.uniqueMemberOverload(cel.StringType, l.uniqueScalar),
 			l.uniqueMemberOverload(cel.BytesType, l.uniqueBytes),
 		),
+		cel.Function("isNan",
+			cel.MemberOverload(
+				"double_is_nan_bool",
+				[]*cel.Type{cel.DoubleType},
+				cel.BoolType,
+				cel.FunctionBinding(func(args ...ref.Val) ref.Val {
+					num, ok := args[0].Value().(float64)
+					if !ok {
+						return types.Bool(false)
+					}
+					return types.Bool(math.IsNaN(num))
+				}),
+			),
+		),
+		cel.Function("isInf",
+			cel.MemberOverload(
+				"double_is_inf_bool",
+				[]*cel.Type{cel.DoubleType},
+				cel.BoolType,
+				cel.FunctionBinding(func(args ...ref.Val) ref.Val {
+					num, ok := args[0].Value().(float64)
+					if !ok {
+						return types.Bool(false)
+					}
+					return types.Bool(math.IsInf(num, 0))
+				}),
+			),
+			cel.MemberOverload(
+				"double_int_is_inf_bool",
+				[]*cel.Type{cel.DoubleType, cel.IntType},
+				cel.BoolType,
+				cel.FunctionBinding(func(args ...ref.Val) ref.Val {
```

Comment:
Wouldnt a `BinaryBinding` be better here?

Reply:
also, is this function necessary?

Reply:
If I can't scope it, might as well match the standard api